runtime/syntax/cake.yaml

```
@@ -4,4 +4,4 @@ detect:
 
 rules:
     - include: "csharp"
-    - preproc: "^[[:space:]]*#(addin|break|l|load|module|r|reference|tool)"
+    - preproc: "^[[:space:]]*#(addin|break|l(oad)?|module|r(reference)?|tool)"
```

I'm guessing this is a typo?

```suggestion
    - preproc: "^[[:space:]]*#(addin|break|l(oad)?|module|r(eference)?|tool)"
```

---

Perhaps correcting this at 2am was not the best idea.